src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -784,58 +784,46 @@ def test_prometheus_config_security_disabled(self, _run_cephadm, cephadm_module:
 
                 scrape_configs:
                   - job_name: 'ceph'
-                    honor_labels: true
                     relabel_configs:
                     - source_labels: [__address__]
                       target_label: cluster
                       replacement: fsid
                     - source_labels: [instance]
                       target_label: instance
                       replacement: 'ceph_cluster'
+                    honor_labels: true
                     http_sd_configs:
                     - url: http://[::1]:8765/sd/prometheus/sd-config?service=mgr-prometheus
 
                   - job_name: 'node'
-                    http_sd_configs:
-                    - url: http://[::1]:8765/sd/prometheus/sd-config?service=node-exporter
```

Comment:
are these changes mandatory? because we are just changing the order ... if no plz keep the previous order